bears/csv/CSVLintBear.py

```diff
@@ -9,6 +9,15 @@
 class CSVLintBear:  # pragma nt: no cover
     """
     Verifies using ``csvlint`` if ``.csv`` files are valid CSV or not.
+    The ``CSVLintBear`` has validation that checks:
+    - structural formatting of a CSV file
+    - delimiter-separated values (dsv) file accessible via URL, File,
+    or an IO-style object (e.g. StringIO)
+    - against CSV dialects
+    - against multiple schema standards; CSV on the Web
```

Comment:
What does `CSV on the web` means ?

Reply:
CSV on the web provides standard ways to express useful metadata about CSV files and other kinds of tabular data.

Reply:
So, it isn't a validation check right ?

Reply:
No

Reply:
Just replace semi-colon with `and` to differentiate between “ schema standards“ and “csv on the web”